modules/monitoring-4-20-release-notes.adoc

```diff
@@ -7,7 +7,9 @@
 = {ocp} {product-version} monitoring release notes
 
 [role="_abstract"]
-The changes for {ocp} {product-version} monitoring are included in the following errata advisory:
+Changes for {ocp} {product-version} monitoring stack, including new features and enhancements, Technology Previews, deprecated and removed features, known issues, and fixed issues.
+
+The changes are included in the following errata advisory:
```

Comment:
Nit: :slightly_smiling_face: IMHO, you could simplify and shorten this. Whatever is in the errata is there, and whatever isn't there isn't there:
```suggestion
Changes for the {ocp} {product-version} monitoring stack are included in the following errata advisory:
```